src/tasks/sep/038-U17-sep-op-sony-ink/config.toml

```diff
@@ -0,0 +1,54 @@
+l2chains = [
+    {name = "OP Sepolia Testnet", chainId = 11155420},
+    {name = "Ink Sepolia Testnet", chainId = 763373},
+    {name = "Soneium Testnet Minato", chainId = 1946}
+]
+
+templateName = "OPCMUpgradeV500"
+
+allowOverwrite = ["SuperchainConfig"]
+
+
+
+# OP Sepolia Testnet (Permissionless)
+
+[[opcmUpgrades]]
+chainId = 11155420
+absolutePrestate = "0x03799051d2bfe459127d4597f469f535ff1bd2a6e1e2134443167620871c11f3"  # prestates."1.8.0-rc.3" https://github.com/ethereum-optimism/superchain-registry/blob/c9881d543174ff00b8f3a9ad3f31bf4630b9743b/validation/standard/standard-prestates.toml#L6
+expectedValidationErrors = "PDDG-VM-30,PLDG-VM-30"
+
+# Soneium Testnet Minato (Permissioned)
+
+[[opcmUpgrades]]
+chainId = 1946
+absolutePrestate = "0x03799051d2bfe459127d4597f469f535ff1bd2a6e1e2134443167620871c11f3"  # prestates."1.8.0-rc.3" https://github.com/ethereum-optimism/superchain-registry/blob/c9881d543174ff00b8f3a9ad3f31bf4630b9743b/validation/standard/standard-prestates.toml#L6
+expectedValidationErrors = "PDDG-VM-30,PDDG-130,PLDG-10"
+
+# Ink Sepolia Testnet (Permissionless)
+
+[[opcmUpgrades]]
+chainId = 763373
+absolutePrestate = "0x03799051d2bfe459127d4597f469f535ff1bd2a6e1e2134443167620871c11f3"  # prestates."1.8.0-rc.3" https://github.com/ethereum-optimism/superchain-registry/blob/c9881d543174ff00b8f3a9ad3f31bf4630b9743b/validation/standard/standard-prestates.toml#L6
+expectedValidationErrors = "PDDG-VM-30,PLDG-VM-30"
+
+
+
+[addresses]
+
+OPCM = "0x6af7ece7d6506be573841b8cf29096638df5fc4f"  # Source: https://www.notion.so/oplabs/op-contracts-release-for-U17-Jovian-294f153ee16280b99539c5b6de67c72f?source=copy_link#295f153ee162800a84c7f37811ecf3ec
```

Comment:
Should link to https://github.com/ethereum-optimism/superchain-registry/blob/7ec506447493842273f45b6d3ca7d347d314debe/validation/standard/standard-versions-sepolia.toml#L23 as the source of truth